app/src/main/res/layout/sheet_medicine_frequency.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:layout_width="match_parent"
+  android:layout_height="wrap_content"
+  android:paddingStart="@dimen/spacing_16"
+  android:paddingEnd="@dimen/spacing_16"
+  android:paddingBottom="@dimen/spacing_16"
+  tools:ignore="RtlSymmetry">
+
+  <TextView
+    android:id="@+id/medicineFrequencyTitleTextView"
+    android:layout_width="0dp"
+    android:layout_height="wrap_content"
+    android:layout_marginTop="16dp"
+    android:layout_marginBottom="70dp"
+    android:textAppearance="@style/Clinic.V2.TextAppearance.H6.Grey0"
+    app:layout_constraintBottom_toTopOf="@+id/radioGroup"
+    app:layout_constraintEnd_toEndOf="parent"
+    app:layout_constraintStart_toStartOf="parent"
+    app:layout_constraintTop_toTopOf="parent"
+    tools:text="Medicine, 10mg" />
+
+  <RadioGroup
+    android:id="@+id/radioGroup"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    app:layout_constraintEnd_toEndOf="parent"
+    app:layout_constraintStart_toStartOf="parent"
+    app:layout_constraintTop_toBottomOf="@+id/medicineFrequencyTitleTextView">
+
+    <com.google.android.material.radiobutton.MaterialRadioButton
+      android:id="@+id/medicineFrequencyOdRadioButton"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:layout_marginTop="16dp"
+      android:padding="@dimen/spacing_16"
+      android:text="@string/sheet_medicine_frequency_od"
+      android:textAppearance="@style/Clinic.V2.TextAppearance.Body1Left.Grey0"
+      android:textColor="@drawable/radio_button_selected" />
+
+    <View
+      android:id="@+id/view"
```

Comment:
Let's rename this divider views id to be more accurate of which views divider they are, for example like `medicineFrequencyOdDivider`.